bedrock/base/templates/macros.html

```diff
@@ -255,6 +255,11 @@ <h3>{{ _('Your download link was sent.') }}</h3>
   </a>
 {%- endmacro %}
 
+<!--
+  monitor_button:
+  - entrypoint must remain last parameter on url for regex in metrics flow js.
+  - form_type value will change on output of https://github.com/mozilla/fxa/issues/963
```

Comment:
Is this something we need to call out in the code, or should we file an issue in GitHub to keep track of it. It seems there is nothing to do right now, so could this get lost / forgotten?

Reply:
Agreed! Filed issue: https://github.com/mozilla/bedrock/issues/7551